### PR TITLE
[dhcp_server] Add rsyslog support for dhcp_server

### PR DESCRIPTION
**Pull request by yaqiangz**
#### Why I did it
Fix issue that dhcp_server doesn't create syslog into host https://github.com/sonic-net/sonic-buildimage/issues/18472

##### Work item tracking
- Microsoft ADO **(number only)**: 28396389

#### How I did it
Modify rsyslog config template

#### How to verify it
UTs passed
Buildimage and install in testbed to verify

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: rules/config [hunk 1]
```diff
@@ -150,7 +150,7 @@ INCLUDE_NAT = y
 INCLUDE_DHCP_RELAY = y
 
 # INCLUDE_DHCP_SERVER - build and install dhcp-server package
-INCLUDE_DHCP_SERVER ?= n
+INCLUDE_DHCP_SERVER ?= y
```
Comment (yaqiangz) [inline rules/config:153]: Remember to restore it when this PR is ready

File: src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf [hunk 2]
```diff
@@ -15,23 +15,14 @@
...
-$SystemLogRateLimitBurst {{ rate_limit_burst }}
-{% endif %}
 
 $ModLoad imklog   # provides kernel logging support
 #$ModLoad immark  # provides --MARK-- message capability
 
 # provides UDP syslog reception
 $ModLoad imudp
-$UDPServerAddress {{udp_server_ip}}  #bind to localhost before udp server run
+$UDPServerAddress 10.150.22.222  #bind to localhost before udp server run
```
Comment (Blueve) [inline src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf:25]: pls remember revert this

File: src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf [hunk 3]
```diff
@@ -93,4 +84,7 @@ $RepeatedMsgReduction on
 # The omfwd plug-in provides the core functionality of traditional message
 # forwarding via UDP and plain TCP. It is a built-in module that does not need
 # to be loaded.
-# TODO rsyslog issue in bridge mode container, don't update to remote server for now
+
+*.*
+action(type="omfwd" Target="10.150.22.222" Port="514" Protocol="udp" Template="SONiCForwardFormat")
```
Comment (Blueve) [inline src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf:89]: pls remember fix the hardcode value

File: platform/broadcom/sai-modules.mk [hunk 4]
```diff
@@ -10,7 +10,7 @@ $(BRCM_OPENNSL_KERNEL)_MACHINE = broadcom
 SONIC_DPKG_DEBS += $(BRCM_OPENNSL_KERNEL)
 
 # SAI bcm modules for DNX family ASIC
```
Comment (yaqiangz) [inline platform/broadcom/sai-modules.mk:12]: Remember to revert it.

File: src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf (hunk 2 above)
Comment (yaqiangz) [reply src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf]: Addressed

File: src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf (hunk 3 above)
Comment (yaqiangz) [reply src/sonic-config-engine/tests/sample_output/py3/rsyslog.conf]: Addressed

**Review by Blueve — APPROVED** [review]

**Comment by yaqiangz** [comment]
Hi @yxieca could you please help to merge this PR?

**Comment by yaqiangz** [comment]
Hi @yxieca could you please help to merge this PR?

**Comment by yaqiangz** [comment]
Hi @bingwang-ms could you please help to approve backport request?

**Pull request merged**